pkg/bootsh

```diff
@@ -0,0 +1,32 @@
+[mirrors]
+https://github.com/davidar/bootsh/archive/refs/tags/v0.2.1.tar.gz
+
+[vars]
+filesize=4533044
+sha512=b0dca90e66dc3e7e3ef9963a15421cac3366d6aeb8cbe1b1ec1c2eba864be58e4dd8cb47cbdd3e1697bd5d6d1219575e5ad443cd781cf261edcb9d121f770aa9
+tarball=bootsh-0.2.1.tar.gz
+pkgver=3
+
+[deps.host]
+bash
```

Comment:
is bash really required ?

Reply:
At the moment, yeah, unfortunately so, the toybox build scripts are full of bashisms. At some point I might rip that all out and replace it with something simpler, but haven't gotten around to it yet

Reply:
Just about to fix this